fiware-region-sanity-tests/commons/http_phonehome_server.py

```diff
@@ -52,13 +52,13 @@ def do_POST(self):
         content_length = int(self.headers['Content-Length'])
         content = self.rfile.read(content_length)
 
-        # Get Hostname from body
+        # Get data from body
         if content:
             dbus_server.emit_phonehome_signal(str(content))
             self.send_response(200)
         else:
             # Bad Request
-            self.send_response(400, message="Invalid hostname received in HTTP PhoneHome request")
+            self.send_response(400, message="Invalid data received in HTTP PhoneHome request")
 
 
 class HttpPhoneHomeServer():
```

Comment:
Perhaps you can use a new URI resource to receive this new info from VM. In this way, we could have different resources:
- /phonehome  -> Emit PhoneHome signal to Dbus.
- /metadata      -> Emit Metadata signal to Dbus
or something like this...